lms/djangoapps/grades/docs/decisions/0001-gradebook-api.rst

```diff
@@ -77,3 +77,10 @@ Decisions
       with no score data available from either ``courseware.StudentModule`` records or from the `Submissions` API.
       In this case, the only score data available exists in the grade override record, and the subsection ``update()``
       call should be forced to read from this record.
+
+   f. We have to synchronously update each grade record for each user in this endpoint. This means the request
```

Comment:
Could you also update part (d) to take out the part about an async task?  And part (e) to take out the part about signals?